cli/cmd/Deploy.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/nais/naisd/api"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Deploys your application",
+	Long:  `Deploys your application`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		clusters := map[string]string{
+			"ci":           "nais-ci.devillo.no",
```

Comment:
Do we want CI here at all?

Reply:
I don't know, is the cluster in use?

Reply:
its part of the nais cluster verifiation process. But then again, its a good thing to verify this client as well. So I change my mind. Keep CI